code/modules/clothing/jewelry/lockets.dm

```diff
@@ -1,14 +1,12 @@
-/obj/item/clothing/accessory/locket
+/obj/item/clothing/jewelry/locket
```

Comment:
Seems a bit weird to me that a locket isn't a type of necklace, but I suppose that's fine. Not blocking. left this comment before I found a blocker.